server/database/config/dbBuild.sql

```diff
@@ -1,77 +0,0 @@
-BEGIN;
```

Comment:
why you delete the schema? if there is any update just push on it and add commit about your changing